libs/core/web/src/lib/pages/login/Login.tsx

```diff
@@ -53,61 +64,62 @@ export function Login() {
       sx={{ minHeight: '100vh' }}
     >
       <Grid item xs={3}>
-        <Typography variant="h2" component="div">
-          Connexion
-        </Typography>
+        <Typography variant="h2">{t(i18nKeys.title.login)}</Typography>
 
-        {!isLoggedIn && (
-          <>
-            <Box component="form" onSubmit={handleSubmit(onSubmit)}>
-              <Grid container direction="column" spacing={1}>
-                <Grid item>
-                  <InputTextField
-                    size={EnumSize.small}
-                    error={errors.email?.message}
-                    id="email"
-                    type="email"
-                    label="Email *"
-                    variant={EnumVariant.outlined}
-                    register={register}
-                  />
-                </Grid>
-                <Grid item>
-                  <InputTextField
-                    size={EnumSize.small}
-                    error={errors.password?.message}
-                    id="password"
-                    type="password"
-                    label="Password *"
-                    variant={EnumVariant.outlined}
-                    register={register}
-                  />
-                </Grid>
-                <Grid item>
-                  <Button variant="outlined" type="submit" fullWidth>
-                    Connexion
-                  </Button>
-                </Grid>
+        <FormProvider {...methods}>
+          <Box
+            component="form"
+            sx={{
+              minWidth: '20rem',
+            }}
+            onSubmit={handleSubmit(onSubmit)}
+          >
+            <Grid container direction="column" spacing={1}>
+              <Grid item>
+                <InputTextField
+                  size={EnumSize.small}
+                  error={
+                    errors?.email?.message ? t(`${errors?.email?.message}`) : ''
+                  }
+                  id="email"
+                  type="email"
+                  label={`${t(i18nKeys.input.label.email)}`}
+                  variant={EnumVariant.outlined}
+                  required={true}
+                />
               </Grid>
-            </Box>
-
-            <Typography
-              gutterBottom
-              variant="body1"
-              sx={{
-                color: '#d32f2f',
-              }}
-            >
-              {error}
-            </Typography>
-          </>
-        )}
+              <Grid item>
+                <InputTextField
+                  size={EnumSize.small}
+                  error={
```

Comment:
handle errors inside `InputTextField`, extract the  error from useController and set the `helperText` prop to this value
```ts
errors?.password?.message
                      ? t(`${errors?.password?.message}`)
                      : ''
```